### DISCUSSION TITLE
Lab-2 c simulation of FIRN11Stream will test fail

### DISCUSSION THREAD

**Discussion by michael61112** [bol-edu/HLS-SOC-Discussions#112] [Q&A]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/9332860/d9e39164-9bdd-49d5-bd79-519ecde3c0b1)
有確認過沒有import錯檔案，單純c simulation會遇到test fail? 不是應該不用動到code的內容?

**Comment by holyuming** [comment]
#include "FIR.h" ??
